queries.sql

```diff
@@ -0,0 +1,34 @@
+/*Queries that provide answers to the questions from all projects.*/
+SELECT *
+FROM animals
+WHERE NAME LIKE '%mon';
+
+SELECT *
```

Comment:
- [ ] this query should only list the name of the animal.

Reply:
Hi @mikethreels 
did you check this query or just see it 
i use like keyword and % to any name ends by mon according to requirements